src/network/bmproto.py

```diff
@@ -102,7 +102,7 @@ def state_bm_header(self):
         return True
 
     def state_bm_command(self):     # pylint: disable=too-many-branches
-        """Process incoming command"""
+        """Expects the presence of command"""
```

Comment:
I think the original docstring is better

Reply:
Okay . Thank you